src/zappy_gui_src/Core/SFML/Interface/Interface.hpp

```diff
@@ -36,13 +37,27 @@ namespace gui {
             /// \param window The window to be set.
             inline void setWindow(std::shared_ptr<sf::RenderWindow> window) {
                 _window = window;
+                _minimap.setWindow(_window);
+            };
+
+            /// \brief Set the map size.
+            /// \param mapSize The map size.
+            inline void setMapSize(const sf::Vector2f &mapSize) {
```

Comment:
I don't know the behaviour of this inline pls, can u explain, i'm not sure of the result